doc/lua_api.txt

```diff
@@ -62,12 +62,13 @@ Where `<gameid>` is unique to each game.
 The game directory can contain the following files:
 
 * `game.conf`, with the following keys:
-    * `name`: Required, human readable name  e.g. `name = Minetest`
+    * `name`: Required. Allows Minetest to determine the game name even if the
```

Comment:
this is not correct - `name` is the same as a mod's `title` for games. Games don't have an equivalent for `name` yet